converter/input/mgologstash/flow_deserializer.go

```diff
@@ -153,21 +275,27 @@ func (f *flowDeserializer) fillFromIPFIXBSONMap(ipfixMap bson.M, outputFlow *Flo
 				return errors.Errorf("could not convert %+v to int", postNaptDestPortIface)
 			}
 		}
-
 	} else {
 		return errors.New("input map must contain key 'netflow.destinationTransportPort'")
 	}
 
 	var flowStart, flowEnd string // RFC3339Nano timestamps
 
+	//Get the Start, and the end times of the flow
 	flowStartMillisIface, flowStartMillisOk := ipfixMap["flowStartMilliseconds"]
 	flowEndMillisIface, flowEndMillisOk := ipfixMap["flowEndMilliseconds"]
 
+	//Also attempt to get the start and end times relative to system init
 	flowStartUptimeMillisIface, flowStartUptimeMillisOk := ipfixMap["flowStartSysUpTime"]
 	flowEndUptimeMillisIface, flowEndUptimeMillisOk := ipfixMap["flowEndSysUpTime"]
+
+	//get the system init time if possible
 	systemInitTimeMillis, systemInitTimeMillisecondsOk := f.ipfixExporterUptimes[host]
+	//If the system init time isn't present or stored use a relative uptime approach
+	systemRelativeMillis, systemRelativeOk := f.ipfixExporterRelUp[host]
```

Comment:
We pull these map values even if the flow has absolute start and end timestamps. Since this code is hot, we may want to restructure this so they are only pulled if case 1 fails. 
I wouldn't worry too much about this.